PHPCI/Languages/lang.en.php

```diff
@@ -113,6 +113,7 @@
                                 (if you cannot add a phpci.yml file in the project repository)',
     'default_branch' => 'Default branch name',
     'allow_public_status' => 'Enable public status page and image for this project?',
+    'archived' => 'Archived',
```

Comment:
I agree about better language management, but actually is not implemented, maybe you can open a different PR about it..